### DISCUSSION TITLE
How to manage multi monitors?

### DISCUSSION THREAD

**Discussion by Asthestarsfalll** [hyprwm/Hyprland#4865] [Q&A] [answered]
Hi, I have 2 monitors eDP-1( my main laptop monitor) and DP-1(external monitor). I want to only enable DP-1 if using both  monitors. And if using eDP-1 individually, it can automatically enable eDP-1. How can I configure this?

**Comment by fedoranvar** [comment, marked as answer]
@Asthestarsfalll 

you can use  [kanshi](https://git.sr.ht/~emersion/kanshi)

**Reply by Asthestarsfalll** [reply]
Thank you!
